### ISSUE TITLE
Shift the formatting toolbar while creating a note at the top of the editor.

### ISSUE THREAD

**Issue by CleverFool77** [publiclab/plots2#5175]
## Description 
The toolbar is at the bottom of the editor which makes it difficult for a user to do the required formatting. In case of a huge article it will lead to a lot of scrolling which is not a good user experience as for every formatting option, the user needs to scroll down.
It's much better to implement the formatting toolbar at the top and make it sticky so that if the user scrolls down, it stays at the top leading to a better access to toolbar and hence a better User Experience.

eg : I tried editing the first note I published. 
![bold](https://user-images.githubusercontent.com/26685258/54602083-a52fbe80-4a66-11e9-80d9-1c358cab8549.gif)

And I had scroll down below to reach the formating toolbar

**Comment by CleverFool77** [comment]
@publiclab/plots2-reviewers Is this issue correct. 
If approved, then I would like to work on this. :)
Thanks !!

**Comment by gauravano** [comment]
@jywarren @rexagod isn't this implemented a while back? Have we reverted it?

**Comment by jywarren** [comment]
Yes, great! I think we did it in rich mode but not markdown mode! Moving this issue over there as we can probably use the same approach. Thanks!
